config.properties

```diff
@@ -0,0 +1,21 @@
+title = RAINBOW BOOKSTORE
+version = 1.0
+implantation = May 2018
```

Comment:
Add this variable somewhere, because you are not currently using it. Maybe in the first screen

Reply:
ok